CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 1.34.0 - 2023-10-24
+- Feature: support `--device-locale` parameter for `maestro start-device`
+- Feature: add `centerElement` parameter for `scrollUntilVisible` to center on element after scrolling finished
+- Feature: add `power` button support for `pressKey` on Android
+- Change: improve command speed for animation heavy screens
```

Comment:
I would word it like this: 
Feature: added tapOn parameter`waitToSettleTimeoutMs` to control how long it waits to move on to the next command. Helpful for animation heavy apps.